models/Asset/Thumbnail/ImageThumbnailTrait.php

```diff
@@ -350,7 +384,7 @@ protected function convertToWebPath(array $pathReference): ?string
      */
     public function getFrontendPath(): string
     {
-        $path = $this->getPath();
+        $path = $this->getPath(['deferredAllowed' => false, 'frontend' => true]);
```

Comment:
```suggestion
        $path = $this->getPath(['deferredAllowed' => true, 'frontend' => true]);
```
is there a reason why it's `false`? it seems that before was not specified, therefore should consider the default `true` instead

Reply:
@bitbirddev any update on this?